queries.sql

```diff
@@ -188,4 +188,124 @@ FROM animals A
 JOIN owners O
 ON A.owner_id = O.id
 GROUP BY O.full_name
-ORDER BY animal DESC;
\ No newline at end of file
+ORDER BY animal DESC;
+
+/*MANY-TO-MANY QUERIES*/
+
+SELECT
+A.name AS animal_name,
+vets.name AS vet_name,
+visits.date_of_visit
+FROM visits
+JOIN animals A
+ON A.id = visits.animal_id
+JOIN vets
+ON vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC;
+
```

Comment:
- [ ] Great job on implementing this query! 🎉 However, according to the guidelines, it should return just one row, indicating the last animal visited. 🐾🗺️ Let's make the necessary adjustments to ensure it functions correctly. 🛠️👍